src/autoscaler/metricscollector/collector/app_streamer_test.go

```diff
@@ -112,50 +111,17 @@ var _ = Describe("AppStreamer", func() {
 					Value:         "67",
 					Timestamp:     222222,
 				}))
-
-				By("collecting and computing throughput")
-				Consistently(database.SaveMetricCallCount).Should(Equal(4))
```

Comment:
why remove the test of collecting throughput metrics? 
Even there is only container metrics, we need to save throughput = 0  periodically